pages/api/v2/projects/[projectId]/webhook.ts

```diff
@@ -0,0 +1,30 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
```

Comment:
I think we can just add a PUT method to `/api/v2/projects/[projectId]`, since webhook now is just a single setting, not a resource.